namui-cfg/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+use lazy_static::lazy_static;
+use proc_macro::TokenStream;
+use rust_cfg_parser::{parse, CfgValue};
+use std::{collections::HashMap, env, process::Command};
+
+static NAMUI_CFG_ENV_PREFIX: &str = "NAMUI_CFG_";
+
+type CfgMap = HashMap<String, String>;
+
+lazy_static! {
+    static ref CFG_VALUES: Vec<CfgValue> = {
+        let mut cfg_map = CfgMap::new();
+        load_cfg_from_namui_cli(&mut cfg_map).unwrap();
+        load_cfg_from_env(&mut cfg_map).unwrap();
+        cfg_map
+            .iter()
+            .map(|(key, value)| match value.is_empty() {
+                true => CfgValue::Name(key.clone()),
+                false => CfgValue::KeyPair(key.clone(), value.clone()),
+            })
+            .collect()
+    };
+}
+
+#[proc_macro_attribute]
+pub fn namui_cfg(attr: TokenStream, item: TokenStream) -> TokenStream {
+    let expression = parse(&format!("cfg({})", attr.to_string().replace("\n", " "))).unwrap();
```

Comment:
If there is newline, parsing error will occur. so remove newline with `attr.to_string().replace("\n", " ")`